Doc/whatsnew/3.13.rst

```diff
@@ -581,6 +581,10 @@ Pending Removal in Python 3.15
   All arguments will be removed from :func:`threading.RLock` in Python 3.15.
   (Contributed by Nikita Sobolev in :gh:`102029`.)
 
+* ``inspect.getargs`` is deprecated in 3.13 and slated for removal in 3.15,
+  instead use ``inspect.signature(types.FunctionType(co, {}))``.
+  (Contributed by Nikita Sobolev in :gh:`108901`.)
```

Comment:
Well, as discussed in https://github.com/python/cpython/pull/108902#discussion_r1317167991: most of the time, people actually have access to the original function, so most of the time, instead of doing `inspect.getargs(fn.__code__)`, they should just do `inspect.signature(fn)`. It's pretty rare that you have access to the code object and _don't_ have access to the original function -- only then should you do `inspect.signature(types.FunctionType(co, {}))`.

I think we should say something like:

```suggestion
* ``inspect.getargs`` is deprecated in 3.13 and slated for removal in 3.15.
  :func:`inspect.signature` is the most accurate way to obtain the signature
  of a callable. If you only have access to a code object, however,
  ``inspect.signature(types.FunctionType(co, {}))`` can be used as a 
  direct replacement for ``inspect.getargs()``.
  (Contributed by Nikita Sobolev in :gh:`108901`.)
```

(And use similar wording elsewhere in this PR)